tests/test_feature_panel.py

```diff
@@ -0,0 +1,364 @@
+from app.database.models import Feature
+import pytest
+import app.internal.features as internal
+import app.routers.features as route
+
+
+@pytest.fixture
+def mock_features():
+    return [
+        {
+            "name": 'test',
+            "route": '/',
+            "description": 'testing',
+            "creator": 'test'
+        }
+    ]
+
+
+def test_create_features_at_startup(mocker, session, mock_features):
+
+    mocker.patch(
+        'app.internal.features.features',
+        mock_features
+    )
+    mocker.patch(
+        'app.internal.features.is_feature_exist_in_db',
+        return_value=False
+    )
+
+    assert internal.create_features_at_startup(session)
+
+
+def test_create_association(session):
+    assert internal.create_association(
+        db=session, feature_id=1, user_id=1, is_enable=False
+    ) is not None
+
+
+def test_get_user_enabled_features(session):
+
+    internal.create_feature(
+        db=session, name='name', route="route",
+        creator='creator', description='description')
+
+    internal.create_association(
+        db=session, feature_id=1, user_id=1, is_enable=True)
```

Comment:
This code is duplicated in the test several times. Place outside of function so it is reusable.